src/components/UserActions/UserActivity.ts

```diff
@@ -275,9 +277,18 @@ export class UserActivity extends Component {
         const li = document.createElement('li');
         li.classList.add(VIEW_EVENT_CLASS);
 
-        const dataElement = document.createElement('div');
+        let dataElement: HTMLElement;
+        if (action.raw.content_id_key && action.raw.content_id_key === CLICKABLE_URI_ID) {
+            const a = document.createElement('a');
```

Comment:
Not sure about the createElement a v.s. div, that could technically break some custom CSS.
What's the motivation behind that?

Reply:
I used <a> because it makes the css rules cleaner, would wrapping the <a> in a <div> be sufficient?

Reply:
It _should_ be alright. @jeremierobert-coveo , wdyt?

Reply:
Typically it's better to style element with classes (but customer often don't follow this rule).